src/ts/scene/ShopScene.ts

```diff
@@ -69,4 +94,21 @@ export default class ShopScene extends Scene
             y: TopMenu.HEIGHT + Card.HEIGHT * 2 + 100
```

Comment:
절대 크기로 위치를 지정하지 말고, 맵과 카드의 상대적인 크기를 통해 위치를 잡아주도록 합시다.
```suggestion
            x: (Card.WIDTH + shopPaper.width - ((Card.WIDTH + 20) * 3) + 20) / 2 ,
            y: shopPaper.getBottomCenter().y - Card.HEIGHT/2 - 100
```